tools/test-runner.sh

```diff
@@ -115,6 +117,9 @@ Script examples:
     Sets dev-nodes and test-hosts to empty lists
     Reruns mam_SUITE
 
+./tools/test-runner.sh --skip-small-tests --skip-check-rpc-nodes --allow-any-rpc --one-node connect
+    Allows any RPCs in the tests, useful if not all nodes are running on the developer machine
+
```

Comment:
The `--one-node` flag, I use it very often, does this mean that now I have to use both together? Under what scenario would I want `--one-node` without `--allow-any-rpc`?

Reply:
ok, I added automatically assumed `--skip-check-rpc-nodes` and `--allow-any-rpc` to `--one-node`.
The reasoning for this new change is:
- if we use test-runner for running just one test, which we often do, we don't need RPC nodes checks thaat much.


> Under what scenario would I want --one-node without --allow-any-rpc?
When you have suite which uses only one node.
Still, sometimes we have suites which use more than one node, but we ok with tests to fail during dev. So, made different behaviour for `--one-node`.


Theoretically, we can extract `TEST_HOSTS` in the code and only check nodes which are provided in `TEST_HOSTS` against required list. And we would only control `TEST_HOSTS` instead.... mmm, maybe it is a good idea actually :D